databases/database-migrations.md

```diff
@@ -19,6 +19,13 @@ Understand how to change the schema of a database table. This includes operation
 
 Making changes to a database is often necessary as an application evolves. While some database changes require creating or deleting tables, others require modifying an existing table (called a database migration).
 
+**Which companies use Relational Database?**
```

Comment:
😎

Reply:
@alodahl I thought since the database migrations are about adding modifying and deleting, so its also the part of the relational database. hence added these companies. Moreover, the companies that use the database must be performing these operations. So you want me to add companies differently. Or not adding companies will also work as I think.